twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -1,437 +1,15 @@
-        },
-        "2.12": {
-            "util_logging": "c0cba01705e9321b3444adcd4a9ce27c2acefd27e14c13b5aec2c318ce1b4fdf",
-            "scrooge_core": "02a6d7cf9fe8d872dfabd20298e4315d677748708e153d8b464fd5abac9a7430",
-            "scrooge_generator": "e7d5da1e3f0e494d3c81a26f44f3e3dc92d7efd757133de8c71758646fd5a833",
-            "util_core": "65bb92e70f95cbbfc640e54a5823a16154eac1a2631dc0211347e085aaa6ed0b",
-        },
-    }
-
-    scala_version_jar_shas = scala_jar_shas[major_version]
-
-    _scala_maven_import_external(
-        name = "io_bazel_rules_scala_scrooge_core",
-        artifact = _scala_mvn_artifact(
-            "com.twitter:scrooge-core:18.6.0",
-            major_version,
-        ),
-        artifact_sha256 = scala_version_jar_shas["scrooge_core"],
-        licenses = ["notice"],
-        server_urls = maven_servers,
-    )
-    native.bind(
-        name = "io_bazel_rules_scala/dependency/thrift/scrooge_core",
-        actual = "@io_bazel_rules_scala_scrooge_core",
-    )
-
-    #scrooge-generator related dependencies
-    _scala_maven_import_external(
-        name = "io_bazel_rules_scala_scrooge_generator",
-        artifact = _scala_mvn_artifact(
-            "com.twitter:scrooge-generator:18.6.0",
-            major_version,
-        ),
-        artifact_sha256 = scala_version_jar_shas["scrooge_generator"],
-        licenses = ["notice"],
-        server_urls = maven_servers,
-    )
-    native.bind(
-        name = "io_bazel_rules_scala/dependency/thrift/scrooge_generator",
-        actual = "@io_bazel_rules_scala_scrooge_generator",
-    )
-
-    _scala_maven_import_external(
-        name = "io_bazel_rules_scala_util_core",
-        artifact = _scala_mvn_artifact(
-            "com.twitter:util-core:18.6.0",
-            major_version,
-        ),
-        artifact_sha256 = scala_version_jar_shas["util_core"],
-        licenses = ["notice"],
-        server_urls = maven_servers,
-    )
-    native.bind(
-        name = "io_bazel_rules_scala/dependency/thrift/util_core",
-        actual = "@io_bazel_rules_scala_util_core",
-    )
-
-    _scala_maven_import_external(
-        name = "io_bazel_rules_scala_util_logging",
-        artifact = _scala_mvn_artifact(
-            "com.twitter:util-logging:18.6.0",
-            major_version,
-        ),
-        artifact_sha256 = scala_version_jar_shas["util_logging"],
-        licenses = ["notice"],
-        server_urls = maven_servers,
-    )
-    native.bind(
-        name = "io_bazel_rules_scala/dependency/thrift/util_logging",
-        actual = "@io_bazel_rules_scala_util_logging",
-    )
-
-def _colon_paths(data):
-    return ":".join([f.path for f in sorted(data)])
-
-ScroogeAspectInfo = provider(fields = [
-    "thrift_info",
-    "src_jars",
-    "output_files",
-    "java_info",
-])
-
-ScroogeInfo = provider(fields = [
-    "aspect_info",
-])
-
-ScroogeImport = provider(fields = [
```

Comment:
this don't exist now right? Can we import `ScroogeImport = ScalaThriftImport` or something so we don't break the provider API? Similarly for the others.

Reply:
👍can do